### PR TITLE
Specify TCP or UDP for the defaults

### PR DESCRIPTION
**Pull request by Loki-101**
It should make it easier for the people who are newer to networking.

# Description

I just updated the Readme.MD file so it specified TCP or UDP for the default ports to make it easier for anyone unfamiliar with networking or the software involved.

## Checklist for all submissions

 *[ X] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by gOOvER** [comment]
ptero opens TCP and UDP and this should be clear, when you have a VPS

**Comment by Loki-101** [comment]
Depends if docker is set up to override the firewall, I've seen a lot of people have to port forward with UFW in addition to a panel allocation.

Definitely not needed, I agree, but it improves the documentation slightly in my opinion.

**Comment by gOOvER** [comment]
> Depends if docker is set up to override the firewall, I've seen a lot of people have to port forward with UFW in addition to a panel allocation.
> 
> Definitely not needed, I agree, but it improves the documentation slightly in my opinion.

Home hosting is not really the sense of Ptero. And people should have basic linux knowledge.

**Comment by Loki-101** [comment]
You don't have to change all the documentation for all the eggs? I changed this one and they don't all have to be uniform.
In my opinion I don't see how more detailed documentation could hurt.

Either way it feels like we're not going to agree on this, so I'll stop debating for now.

**Comment by gOOvER** [comment]
> You don't have to change all the documentation for all the eggs? I changed this one and they don't all have to be uniform. In my opinion I don't see how more detailed documentation could hurt.
> 
> Either way it feels like we're not going to agree on this, so I'll stop debating for now.

you have; why doingg the work yor linux noobs, sry

**Comment by Loki-101** [comment]
My reasoning is that when I use a new egg I look at the instructions that come with it, and I'm a bit of a perfectionist. If I see something that can be improved and there is a convenient edit button I can't resist.

More than that, this is my first time contributing with GitHub. I'm new to the interface and wanted to use this as a kind of test run because it seemed harmless enough and likely to get approved. I realize people can just search "Geiser Floodgate Firewall Rules" and I'm not advocating them being lazy, but I just don't see any harm in this.

**Comment by gOOvER** [comment]
> My reasoning is that when I use a new egg I look at the instructions that come with it, and I'm a bit of a perfectionist. If I see something that can be improved and there is a convenient edit button I can't resist.
> 
> More than that, this is my first time contributing with GitHub. I'm new to the interface and wanted to use this as a kind of test run because it seemed harmless enough and likely to get approved. I realize people can just search "Geiser Floodgate Firewall Rules" and I'm not advocating them being lazy, but I just don't see any harm in this.

noone read the readme ;)

File: game_eggs/minecraft/crossplay/purpur-geysermc-floodgate/README.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ Purpur-GeyserMC-Floodgate is a drop-in replacement for Paper servers designed fo
 
 ## Server Ports
 
-The minecraft server requires a single port for access (default 25565). GeyserMC requires an additional port (default 19132). Any other plugins you add may require extra ports to enabled for the server.
+The minecraft server requires a single port for access (default 25565 TCP). GeyserMC requires an additional port (default 19132 UDP). Any other plugins you add may require extra ports to enabled for the server.
```
Comment (QuintenQVD0) [inline game_eggs/minecraft/crossplay/purpur-geysermc-floodgate/README.md:14]: No need to add the protocol here. In the table fine not here.

**Review by QuintenQVD0 — CHANGES_REQUESTED** [review]

File: game_eggs/minecraft/crossplay/purpur-geysermc-floodgate/README.md (hunk 1 above)
Comment (Loki-101) [reply game_eggs/minecraft/crossplay/purpur-geysermc-floodgate/README.md]: Fixed.

**Review by QuintenQVD0 — APPROVED** [review]

**Comment by QuintenQVD0** [comment]
LGTM! thx

**Pull request merged**